src/features/presence/PresenceCollaborator.js

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import classnames from 'classnames';
+import isEmpty from 'lodash/isEmpty';
+import PropTypes from 'prop-types';
+import { FormattedMessage, injectIntl } from 'react-intl';
+
+import Link from '../../components/link/LinkBase';
+import messages from './messages';
+import PresenceAvatar from './PresenceAvatar';
+import { determineInteractionMessage } from './utils/presenceUtils';
+
+import './PresenceCollaborator.scss';
+
+export const renderTimestampMessage = (interactedAt, interactionType, intl) => {
+    const lastActionMessage = determineInteractionMessage(interactionType, interactedAt);
+    const timeAgo = intl.formatRelativeTime
+        ? intl.formatRelativeTime(interactedAt - Date.now())
+        : intl.formatRelative(interactedAt);
+
+    if (lastActionMessage) {
+        return (
+            <FormattedMessage
+                {...lastActionMessage}
+                values={{
+                    timeAgo,
+                }}
+            />
+        );
+    }
+    return null;
+};
+
+const PresenceCollaborator = ({ collaborator, disableHover, intl }) => {
+    const { avatarUrl, id, interactedAt, interactionType, isActive, name, profileUrl } = collaborator;
+
+    const listItemClasses = classnames('bdl-PresenceCollaborators-item', {
```

Comment:
Let's rename these classes with `bdl-PresenceCollaborator`, as the plural and `-item` suffix are not needed.